lib/expo/message.ex

```diff
@@ -24,9 +24,7 @@ defmodule Expo.Message do
 
   See `key/1`.
   """
-  @opaque key ::
-            {msgctxt :: String.t(),
-             msgid :: String.t() | {msgid :: String.t(), msgid_plural :: String.t()}}
+  @opaque key :: Singular.key() | Plural.key()
```

Comment:
@whatyouhide The type `Expo.Message.key/0` was already opaque. The return type of the `Expo.Message.{Singular,Plural}.key/1` however was not.

I thought it makes sense to declare it as opaque at the source so that changing the contents will not be considered a breaking change in the future.